### PR TITLE
Fix for gallery image rotation

### PR DESCRIPTION
**Pull request by tikank**
Fix for issue #2418. Rotate is replaced with transpose which sets image size correctly.

### CODE REVIEW

File: nikola/image_processing.py [hunk 1]
```diff
@@ -117,13 +117,13 @@ def resize_image(self, src, dst, max_size, bigger_panoramas=True, preserve_exif_
...
+                im = im.transpose(Image.ROTATE_180)
             elif value in (5, 6):
-                im = im.rotate(270)
+                im = im.transpose(Image.ROTATE_270)
             elif value in (7, 8):
-                im = im.rotate(90)
+                im = im.transpose(Image.ROTATE_90)
             if value in (2, 4, 5, 7):
-                im = ImageOps.mirror(im)
+                im = im.transpose(Image.FLIP_LEFT_RIGHT)
```
Comment (ralsina) [inline nikola/image_processing.py]: This means you can delete the ImageOps import, I think (Travis tests will fail if you don't)

**Comment by ralsina** [comment]
LGTM, could you add a 1-liner in CHANGES and add yourself to AUTHORS?

**Comment by tikank** [comment]
Added.

**Comment by ralsina** [comment]
Cool! Thanks for the contribution!

**Pull request merged**